app/database/models/ms_schema/tasks_list.py

```diff
@@ -0,0 +1,210 @@
+from enum import unique, Enum
+from datetime import date
+from app.database.db_types.JsonCustomType import JsonCustomType
+from app.database.sqlalchemy_extension import db
+
+
+class TasksListModel(db.Model):
+    """Model representation of a list of tasks.
+    
+    Attributes:
+        id: Id of the list of tasks.
+        tasks: A list of tasks, using JSON format.
+        next_task_id: Id of the next task added to the list of tasks.
+    """
+
+    __tablename__ = "tasks_list"
+    __table_args__ = {"schema": "public", "extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+    tasks = db.Column(JsonCustomType)
+    next_task_id = db.Column(db.Integer)
+
+    def __init__(self, tasks: "TasksListModel" = None):
+        """Initializes tasks.
+
+        Args:
+            tasks: A list of tasks.
+        
+        Raises:
+            A Value Error if the task is not initialized.
+        """
+
+        if tasks is None:
+            self.tasks = []
+            self.next_task_id = 1
+        else:
+            if isinstance(tasks, list):
+                self.tasks = []
+                self.next_task_id = len(tasks) + 1
+            else:
+                raise ValueError(TypeError)
```

Comment:
I don't understand this, why are errors nested?

Reply:
I'm not sure tbh. All the mentorship system models in this schema (ms_schema) are directly copy pasted from mentorship code without any change (except for adding db.relation to connect them to bit tables).
I was under the assumption that all codes there (mentorship system) have been passed checks and reviews by others since its creation, hence didn't feel the need to double check or concern on why a particular style is chosen (I was focusing more on the BIT code side so far that I wrote myself). If you think we should double check and change the mentorship code in BIT side,I'll do that too.